merlin-server/sql/merlin/tables/plan.sql

```diff
@@ -8,6 +8,7 @@ create table plan (
 
   -- TODO: Remove 'start_time'; its purpose will be served by a model-specific entry in 'simulation.arguments'.
   start_time timestamptz not null,
+  parent_id integer,
```

Comment:
key constraint pls

Reply:
In this case with a key constraint it would be nice to explicitly say `parent_id integer null` just to be clear.

Reply:
This can't be a foreign key column because it can be null

Reply:
I did a little research before, and I thought it could be. It only enforces the constraint if it is not null. I haven't tried it myself though.

Reply:
CREATE TABLE some_child (
    parent_id integer references parent(id) on delete set null
);

Reply:
I think we ought to decide as a team if we want to orphan the children of deleted plans. The other two options are 

1. keep the parent_id and preserve the relationship between plans
2. have the grandparent plan 'adopt' the immediate children of deleted plans, which preserves the list of which plans can merge into each other, if not the relationship between them

Reply:
The adoption strategy has been implemented.